meta-mender-core/classes/mender-maybe-setup.bbclass

```diff
@@ -68,6 +68,9 @@ python() {
 
         # Enable the testing/* layers and functionality
         'mender-testing-enabled',
+
+        # Enable installation of the Mender-client network scripts
+        'mender-inventory-network-scripts'
```

Comment:
I think you misunderstood, we should not use `MENDER_FEATURES` at all for this. `MENDER_FEATURES` are for features that affect the global build. So this, and all other references in class files, should be removed. See other comments for how to do it with `PACKAGECONFIG`.

Reply:
I think this was my bad, then. I misunderstood too...